src/main/java/frc/robot/OI.java

```diff
@@ -64,6 +69,8 @@ private OI() {
         secondLevelUnready.whenPressed(new InstantCommand(Climber.getInstance(), () -> Climber.getInstance().setFrontFoot(DoubleSolenoid.Value.kForward)));
         boardButtons[RobotMap.OI.SECOND_LEVEL_UNREADY] = secondLevelUnready;
         preclimb.whenPressed(new ToggleRearFeet());
+
+        winchClimber.whileHeld(new WinchesForward());
```

Comment:
Can you move this to keep all the stuff with `preclimb` in one continuous block?